src/Api/PubnubApi/EventEngine/Subscribe/Events/SubscriptionEvents.cs

```diff
@@ -40,11 +40,15 @@ public class HandshakeReconnectGiveUpEvent : Core.IEvent {
 	}
 
 	public class ReceiveSuccessEvent : Core.IEvent {
-		public List<PNMessageResult<object>> messages;
-		public SubscriptionCursor cursor;
+		public IEnumerable<string> Channels;
```

Comment:
per CE decisions document, `RECEIVE_SUCCESS` event do not have `Channels` and `ChannelGroups`. If we don't have, how can ReceiveState get info of `Channels` and `ChannelGroups`, which expects the data from `ReceiveSuccessEvent`

Reply:
Probably Channels and ChannelGroups are declared in global context.

Reply:
Agreed, the document is missing that.